ui/src/app/routing/guards.tsx

```diff
@@ -90,15 +89,6 @@ export const adminLockedGuard = (ns: string, wsid: string): Guard => {
   };
 };
 
-// Temp: Since the new analysis page is not ready yet, use the guard to prevent users to go to
-// new analysis page, if the flag is off
-export const tempAppsAnalysisGuard = (ns: string, wsid: string): Guard => {
```

Comment:
no longer needed, because there is no longer a distinct route for the new tab.  We only render one or the other.